BuildDetails.md

```diff
@@ -0,0 +1,39 @@
+This is the build description guide that explains the build process for all FullScreenShenanigans projects. The guide will explain to you how the project is built and why it is built that way.
```

Comment:
Note that FullScreenShenanigans projects include modules. This build guide only *fully* applies to the GameStartr games, though large parts of it apply to the modules as well.